plugins/BEdita/API/tests/TestCase/Controller/AdminControllerTest.php

```diff
@@ -69,11 +69,17 @@ public function testIndex()
                 ],
             ]
         ];
-
-        $this->configRequestHeaders();
+        // GET /admin with anonymous user
+        $this->configRequestHeaders('GET');
+        $this->get('/admin/applications');
+        $result = json_decode((string)$this->_response->getBody(), true);
+        static::assertResponseCode(401);
+        static::assertContentType('application/vnd.api+json');
+        // GET /admin with authenticated user
+        $authHeader = $this->getUserAuthHeader();
+        $this->configRequestHeaders('GET', $authHeader);
```

Comment:
a testcase of a user logged with no administrator role is needed - response must be 403